docs(manage_infrastructure): document usage-report plugin

thomas-dkmt opened the pull request:
The **usage-report** plugin was not covered at all in the Xen Orchestra technical documentation.

This improvement explains how to get usage reports from your resources, by enabling and setting up the plugin, in the [**Management → Infrastructure management → Health**](https://docs.xen-orchestra.com/manage_infrastructure#health) section of the XO documentation.

thomas-dkmt committed:
doc(XO): Document the usage-reports plugin

The usage-reports plugin was not covered at all
in the Xen Orchestra technical documentation.

This improvement explains how to get usage reports from your resources, by enabling and setting up the plugin

Signed-off-by: Thomas Moraine <thomas.moraine@vates.tech>

lsouai-vates commented:
related to: https://project.vates.tech/vates-global/browse/XO-984/

lsouai-vates commented on docs/docs/manage_infrastructure.md:
```diff
@@ -753,6 +753,25 @@ This is the place to compare metrics on comparable objects (VMs to VMs, hosts to
+A list of plugins appears.
+2. From the list, look for the **usage-report** plugin. You can scroll through the list or type the plugin name in the search bar.
+3. Enable the plugin by activating the toggle switch next to the plugin name.
+4. Click the **-** icon to reveal the plugin settings :
```
```suggestion
4. Click the **+** icon to reveal the plugin settings :
```

thomas-dkmt committed:
Update docs/docs/manage_infrastructure.md

Co-authored-by: Laure S. <laure.souai@vates.tech>

julien-f committed:
Update manage_infrastructure.md

- use `1.` for all list items to make changes either due to autonumbering
- either add a blank to make it a proper paragraph or remove new line character
- avoid forced new lines
